src/main/java/uk/gov/hmcts/divorce/citizen/notification/SaveAndSignOutNotificationHandler.java

```diff
@@ -30,13 +38,28 @@ public void notifyApplicant(State state, CaseData caseData, Long caseId, String
         User user = idamService.retrieveUser(userToken);
         boolean isApplicant1 = ccdAccessService.isApplicant1(userToken, caseId);
 
-        final var applicant = isApplicant1 ? caseData.getApplicant1() : caseData.getApplicant2();
-        final var partner = isApplicant1 ? caseData.getApplicant2() : caseData.getApplicant1();
+        Applicant applicant1 =  caseData.getApplicant1();
+        Applicant applicant2 =  caseData.getApplicant2();
+        final var applicant = isApplicant1 ? applicant1 : applicant2;
+        final var partner = isApplicant1 ? applicant2 : applicant1;
+        final var isInterimApplication = State.AosOverdue.equals(state) && !isEmpty(applicant1.getInterimApplicationOptions());
 
-        final var emailTemplate = State.InformationRequested.equals(state) ? REQUEST_FOR_INFORMATION_SAVE_SIGN_OUT : SAVE_SIGN_OUT;
+        final var emailTemplate = State.InformationRequested.equals(state)
+            ? REQUEST_FOR_INFORMATION_SAVE_SIGN_OUT : isInterimApplication ? INTERIM_APPLICATION_SAVE_SIGN_OUT : SAVE_SIGN_OUT;
         final var templateContent = commonContent.mainTemplateVars(caseData, caseId, applicant, partner);
         templateContent.put(SMART_SURVEY, commonContent.getSmartSurvey());
 
+        if (isInterimApplication) {
+            switch (applicant1.getInterimApplicationOptions().getInterimApplicationType()) {
+                case DEEMED_SERVICE -> templateContent.put(IS_DEEMED_SERVICE, "deemed service");
+                case BAILIFF_SERVICE -> templateContent.put(IS_BAILIFF_SERVICE, "bailiff service");
+                case ALTERNATIVE_SERVICE -> templateContent.put(IS_ALTERNATIVE_SERVICE, "alternative service");
+                case DISPENSE_WITH_SERVICE -> templateContent.put(IS_DISPENSE_SERVICE, "dispense with service");
+                // We could have it as a case or leave it as default for search gov records
+                default -> templateContent.put(IS_SEARCH_GOV_SERVICE, "search government records");
+            }
+        }
```

Comment:
It looks like we could use the enum label directly here and have an application type variable in the template?:
```suggestion
        if (isInterimApplication) {
          templateContent.put(
            INTERIM_APPLICATION_TYPE,
            applicant1.getInterimApplicationOptions().getInterimApplicationType().getLabel()
          )
        }
```